python/src/ir.cc

```diff
@@ -492,13 +496,23 @@ void init_triton_ir(py::module &&m) {
              self->print(os, printingFlags);
              return str;
            })
+      .def("str_nodebug",
```

Comment:
What does `nodebug` refer to here? Does `__str__` add extra debug info?

Reply:
Yeah, it prints the module with the debug locations